### PR TITLE
Update velocity event.mdx

### PR DESCRIPTION
**Pull request by KastenKlicker**
Update event.mdx to remove deprecated velocity API usage.

### CODE REVIEW

**Review by olijeffers0n — APPROVED** [review]

File: docs/velocity/dev/api/event.mdx [hunk 1]
```diff
@@ -33,20 +33,21 @@ and _not_ in `com.google.common.eventbus`.
 
 ## Orders
 
-Every listener has a <Javadoc name={"com.velocitypowered.api.event.PostOrder"} project={"velocity"}>`PostOrder`</Javadoc>.
-When an event is fired, the order in which listeners are invoked is defined by their `PostOrder`.
-Listeners using `PostOrder.FIRST` are called first, then `EARLY`, `NORMAL`, etc.
+Every listener has a <Javadoc name={"com.velocitypowered.api.event.Subscribe"} project={"velocity"}>`priority`</Javadoc>.
```
Comment (powercasgamer) [inline docs/velocity/dev/api/event.mdx:36]:
```suggestion
Every listener has a <Javadoc name={"com.velocitypowered.api.event.Subscribe#priority()"} project={"velocity"}>`priority`</Javadoc>.
```

File: docs/velocity/dev/api/event.mdx [hunk 2]
```diff
@@ -33,20 +33,21 @@ and _not_ in `com.google.common.eventbus`.
...
-@Subscribe(order = PostOrder.NORMAL)
+@Subscribe(priority = 0, order = PostOrder.CUSTOM)
 public void onPlayerChat(PlayerChatEvent event) {
 	// do stuff
 }
 ```
 
-`NORMAL` is the default value if you do not specify an order.
+`-32768` is the default value if you do not specify an order.
+Note that due to compatibility constraints, you must specify <Javadoc name={"com.velocitypowered.api.event.PostOrder"} project={"velocity"}>`PostOrder.CUSTOM`</Javadoc> in order to use this field.
```
Comment (powercasgamer) [inline docs/velocity/dev/api/event.mdx:50]:
```suggestion
:::note

Due to compatibility constraints, you must specify <Javadoc name={"com.velocitypowered.api.event.PostOrder"} project={"velocity"}>`PostOrder.CUSTOM`</Javadoc> in order to use this field.

:::
```
or maybe a warning 🤷

**Review by powercasgamer — CHANGES_REQUESTED** [review]

File: docs/velocity/dev/api/event.mdx [hunk 3]
```diff
@@ -33,20 +33,25 @@ and _not_ in `com.google.common.eventbus`.
...
 public void onPlayerChat(PlayerChatEvent event) {
 	// do stuff
 }
 ```
 
-`NORMAL` is the default value if you do not specify an order.
+`-32768` is the default value if you do not specify an order.
+:::note
+
+Due to compatibility constraints, you must specify <Javadoc name={"com.velocitypowered.api.event.PostOrder"} project={"velocity"}>`PostOrder.CUSTOM`</Javadoc> in order to use this field.
```
Comment (zlataovce) [inline docs/velocity/dev/api/event.mdx:52]:
```suggestion
Due to compatibility constraints, you must specify <Javadoc name={"com.velocitypowered.api.event.PostOrder#CUSTOM"} project={"velocity"}>`PostOrder.CUSTOM`</Javadoc> in order to use this field.
```

File: docs/velocity/dev/api/event.mdx [hunk 4]
```diff
@@ -121,12 +126,12 @@ return an <Javadoc name={"com.velocitypowered.api.event.EventTask"} project={"ve
 or add a second return an <Javadoc name={"com.velocitypowered.api.event.Continuation"} project={"velocity"}>`Continuation`</Javadoc> parameter:
 
 ```java
-  @Subscribe(order = PostOrder.EARLY)
+  @Subscribe(priority = 100,order = PostOrder.CUSTOM)
```
Comment (zlataovce) [inline docs/velocity/dev/api/event.mdx:129]:
```suggestion
  @Subscribe(priority = 100, order = PostOrder.CUSTOM)
```

File: docs/velocity/dev/api/event.mdx [hunk 5]
```diff
@@ -121,12 +126,12 @@ return an <Javadoc name={"com.velocitypowered.api.event.EventTask"} project={"ve
...
   public void onLogin(LoginEvent event, Continuation continuation) {
     doSomeAsyncProcessing().addListener(continuation::resume, continuation::resumeWithException);
   }
 
-  @Subscribe(order = PostOrder.EARLY)
+  @Subscribe(priority = 100,order = PostOrder.CUSTOM)
```
Comment (zlataovce) [inline docs/velocity/dev/api/event.mdx:134]:
```suggestion
  @Subscribe(priority = 100, order = PostOrder.CUSTOM)
```

**Review by zlataovce — CHANGES_REQUESTED** [review]

**Review by zlataovce — APPROVED** [review]

**Pull request merged**